src/bindings/python/tests/test_graph/test_core.py

```diff
@@ -392,6 +392,23 @@ def test_discrete_type_info():
     assert n3.get_type_info().name < n1.get_type_info().name
     assert n3.get_type_info().name <= n1.get_type_info().name
 
+@pytest.mark.parametrize("input_data", [
+    [1, 2, 3],
+    (1, 2, 3),
+])
+def test_shape_equality(input_data):
+    data = input_data
+    s = Shape(data)
+    assert data == s
```

Comment:
```suggestion
])
@pytest.mark.parametrize("shape_type", [Shape, PartialShape])
def test_shape_equality(input_data, shape_type):
    data = input_data
    shape = shape_type(data)
    assert data == shape
```
I'd also prefer to keep one test (as they are the same) and parametrize shape type like this.